test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/TestFiles/Input/ModelExpressionTagHelper.cshtml

```diff
@@ -1,6 +1,9 @@
 @model DateTime
 
 @addTagHelper Microsoft.AspNetCore.Mvc.Razor.InputTestTagHelper, Microsoft.AspNetCore.Mvc.Razor.Host.Test
+@addTagHelper Microsoft.AspNetCore.Mvc.Razor.DictionaryPrefixTestTagHelper, Microsoft.AspNetCore.Mvc.Razor.Host.Test
 
 <input-test for="Now" />
-<input-test for="@Model" />
\ No newline at end of file
+<input-test for="@Model" />
+
+<div prefix-test="@Model" ></div>
```

Comment:
Please add another element using a `DateTime` property and adding more than one entry to the dictionary e.g. `<span prefix-hour="Hour" prefix-minute="Minute">Text</span>`.